pkg/tests/tasks/security/authorization/http_test.go

```diff
@@ -16,105 +16,216 @@ package authorization
 
 import (
 	"fmt"
-	"io/ioutil"
-	"strings"
 	"testing"
-	"time"
 
-	"github.com/maistra/maistra-test-tool/pkg/examples"
-	"github.com/maistra/maistra-test-tool/pkg/util"
-	"github.com/maistra/maistra-test-tool/pkg/util/log"
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	Curl "github.com/maistra/maistra-test-tool/pkg/util/curl"
```

Comment:
Why is the `Curl` alias needed? It works just fine without it.

Reply:
See my last commit message. lint job failed if I don't rename my import .
the "curl" method has been declared in the same package from your deny_test.go file.

Reply:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/maistra_maistra-test-tool/414/pull-ci-maistra-maistra-test-tool-maistra-2.4-lint-2-4/1644073927753863168/artifacts/test/build-log.txt

Reply:
Will be fixed by https://github.com/maistra/maistra-test-tool/pull/415